### DISCUSSION TITLE
Issue with merge audio flac files

### DISCUSSION THREAD

**Discussion by Slavomir9** [mifi/lossless-cut#1729] [Support]
Hello,
I have bug report
If I merge audio flac files, duration of audio file in VLC player is the same as first file. I use Linux Ubuntu.
If I create video from audio, and upload on Youtube, it not work correct. Duration of audio in video is not full

On Linux Ubuntu for me correct work only this command:
`sox part1.flac part2.flac output.flac`

Example of audio flac files for testing
https://www.uschovna.cz/en/zasilka/MUWPRTH69M45V6HF-CYK/RZMVLTT5NK/?set_lang=en

Thank you very much for help or for developing :)

**Comment by mifi** [comment]
hi, isn't it this issue? https://github.com/mifi/lossless-cut/blob/master/issues.md#cut-file-has-same-length-as-input

**Reply by Slavomir9** [reply]
Thank you very much for answer. 
Cut and merge flac file will probably different principle. It is not the same bug.

(Now I tested cut flac files too, and work with the same bug as is described),
